frontend/src/lib/taxonomy.tsx

```diff
@@ -755,6 +758,39 @@ export const KEY_MAPPING: KeyMappingInterface = {
     },
 }
 
+// Creates PROPERTY_NAME_ALIASES in posthog/api/property_definition.py
+// console.log(
+//     JSON.stringify(
+//         Object.fromEntries(
+//             Array.from(Object.entries(KEY_MAPPING.event))
+//                 .map(([key, value]) => [key, value.label])
+//                 .filter(([key, label]) => {
+//                     if (!key) {
+//                         return false
+//                     }
+//                     if (!label || label.includes('deprecated')) {
+//                         return false
+//                     }
+//
+//                     const keyLower = key.toLowerCase()
+//                     const labelLower = label.toLowerCase()
+//
+//                     if (keyLower.includes(labelLower)) {
+//                         return false
+//                     }
+//
+//                     const keyLowerNoSpecial = keyLower.replace(/[$_]+/g, ' ')
+//                     const labelWords = labelLower.split(/\s+/)
+//
+//                     return !labelWords.every((word) => keyLowerNoSpecial.includes(word))
+//                 })
+//                 .sort()
+//         ),
+//         null,
+//         2
+//     )
+// )
+
```

Comment:
Let's not commit in code that's commented out, but delete it instead

Reply:
I don't have strong opinions either way, but keeping it would make updating the list easier in future if we change the labels

Reply:
In this one case it might be useful to leave this in, since it's more of a script for later than commented-out product code

Reply:
Moved to frontend/scripts